terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT ERROR: Rank 141(g:141) received signal SIGABRT/SIGIOT(6).
	Process ID: 68892, Host: r147i5n1, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024
	MPT Version: HPE MPT 2.25  08/14/21 03:20:21

MPT: --------stack traceback-------
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT ERROR: Rank 115(g:115) received signal SIGABRT/SIGIOT(6).
	Process ID: 17147, Host: r147i4n35, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024
	MPT Version: HPE MPT 2.25  08/14/21 03:20:21

MPT: --------stack traceback-------
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT ERROR: Rank 169(g:169) received signal SIGABRT/SIGIOT(6).
	Process ID: 23792, Host: r147i5n3, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024
	MPT Version: HPE MPT 2.25  08/14/21 03:20:21

MPT: --------stack traceback-------
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT: Attaching to program: /proc/68892/exe, process 68892
MPT: [Thread debugging using libthread_db enabled]
MPT: Using host libthread_db library "/lib64/libthread_db.so.1".
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: 0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: warning: File "/nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py" auto-loading has been declined by your `auto-load safe-path' set to "$debugdir:$datadir/auto-load:/usr/bin/mono-gdb.py".
MPT: To enable execution of this file add
MPT: 	add-auto-load-safe-path /nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: To completely disable this security protection add
MPT: 	set auto-load safe-path /
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: For more information about this security protection see the
MPT: "Auto-loading safe path" section in the GDB manual.  E.g., run from the shell:
MPT: 	info "(gdb)Auto-loading safe path"
MPT: Missing separate debuginfos, use: debuginfo-install glibc-2.17-326.el7_9.x86_64 libX11-1.6.7-4.el7_9.x86_64 libXau-1.0.8-2.1.el7.x86_64 libXext-1.3.3-3.el7.x86_64 libglvnd-1.0.1-0.8.git5baa1e5.el7.x86_64 libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.x86_64 libibverbs-41mlnx1-OFED.4.9.3.0.0.49417.x86_64 libmlx5-41mlnx1-OFED.5.0.0.4.0.50218.x86_64 libnl3-3.2.28-4.el7.x86_64 libnuma-3.0sgi-sgi716r61.rhel73.x86_64 libxcb-1.13-1.el7.x86_64
MPT: (gdb) #0  0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: #1  0x00002aaaab01f3b6 in mpi_sgi_system (
MPT: #2  MPI_SGI_stacktraceback (
MPT:     header=header@entry=0x7fffffffad50 "MPT ERROR: Rank 141(g:141) received signal SIGABRT/SIGIOT(6).\n\tProcess ID: 68892, Host: r147i5n1, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024\n\tMPT Version: HPE MPT 2.25  "...) at sig.c:340
MPT: #3  0x00002aaaab01f5b6 in first_arriver_handler (signo=signo@entry=6, 
MPT:     stack_trace_sem=stack_trace_sem@entry=0x2aab26fe0080) at sig.c:489
MPT: #4  0x00002aaaab01f843 in slave_sig_handler (signo=6, 
MPT:     siginfo=<optimized out>, extra=<optimized out>) at sig.c:565
MPT: #5  <signal handler called>
MPT: #6  0x00002aaaef3f1387 in raise () from /lib64/libc.so.6
MPT: #7  0x00002aaaef3f2a78 in abort () from /lib64/libc.so.6
MPT: #8  0x00002aaaeeb6d369 in __gnu_cxx::__verbose_terminate_handler ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/vterminate.cc:95
MPT: #9  0x00002aaaeeb78923 in __cxxabiv1::__terminate(void (*)()) ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:48
MPT: #10 0x00002aaaeeb78968 in std::terminate ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:58
MPT: #11 0x00002aaaeeb78b4d in __cxxabiv1::__cxa_throw (obj=<optimized out>, 
MPT:     tinfo=0x2aaaee8aa4a0 <typeinfo for cv::Exception>, 
MPT:     dest=0x2aaaeda45eb0 <cv::Exception::~Exception()>)
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_throw.cc:95
MPT: #12 0x00002aaaed815e77 in cv::error(cv::Exception const&) [clone .cold] ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #13 0x00002aaaeda4819d in cv::error(int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, char const*, char const*, int) () from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #14 0x00002aaaed9503f8 in cv::operator==(cv::Mat const&, cv::Mat const&) ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #15 0x000000000041e70e in matsnd(cv::Mat const&, int, int) ()
MPT: #16 0x000000000041f6c7 in main ()
MPT: (gdb) A debugging session is active.
MPT: 
MPT: 	Inferior 1 [process 68892] will be detached.
MPT: 
MPT: Quit anyway? (y or n) [answered Y; input not from terminal]
MPT: Detaching from program: /proc/68892/exe, process 68892
MPT: [Inferior 1 (process 68892) detached]
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'


MPT: -----stack traceback ends-----
MPT: On host r147i5n1, Program /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024, Rank 141, Process 68892: Dumping core on signal SIGABRT/SIGIOT(6) into directory /home5/rrramir2/job_scripts/cpu_jobs/1024_images
MPT: Attaching to program: /proc/17147/exe, process 17147
MPT: [Thread debugging using libthread_db enabled]
MPT: Using host libthread_db library "/lib64/libthread_db.so.1".
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: 0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: warning: File "/nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py" auto-loading has been declined by your `auto-load safe-path' set to "$debugdir:$datadir/auto-load:/usr/bin/mono-gdb.py".
MPT: To enable execution of this file add
MPT: 	add-auto-load-safe-path /nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: To completely disable this security protection add
MPT: 	set auto-load safe-path /
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: For more information about this security protection see the
MPT: "Auto-loading safe path" section in the GDB manual.  E.g., run from the shell:
MPT: 	info "(gdb)Auto-loading safe path"
MPT: Missing separate debuginfos, use: debuginfo-install glibc-2.17-326.el7_9.x86_64 libX11-1.6.7-4.el7_9.x86_64 libXau-1.0.8-2.1.el7.x86_64 libXext-1.3.3-3.el7.x86_64 libglvnd-1.0.1-0.8.git5baa1e5.el7.x86_64 libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.x86_64 libibverbs-41mlnx1-OFED.4.9.3.0.0.49417.x86_64 libmlx5-41mlnx1-OFED.5.0.0.4.0.50218.x86_64 libnl3-3.2.28-4.el7.x86_64 libnuma-3.0sgi-sgi716r61.rhel73.x86_64 libxcb-1.13-1.el7.x86_64
MPT: (gdb) #0  0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: #1  0x00002aaaab01f3b6 in mpi_sgi_system (
MPT: #2  MPI_SGI_stacktraceback (
MPT:     header=header@entry=0x7fffffffad50 "MPT ERROR: Rank 115(g:115) received signal SIGABRT/SIGIOT(6).\n\tProcess ID: 17147, Host: r147i4n35, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024\n\tMPT Version: HPE MPT 2.25 "...) at sig.c:340
MPT: #3  0x00002aaaab01f5b6 in first_arriver_handler (signo=signo@entry=6, 
MPT:     stack_trace_sem=stack_trace_sem@entry=0x2aab26fe0080) at sig.c:489
MPT: #4  0x00002aaaab01f843 in slave_sig_handler (signo=6, 
MPT:     siginfo=<optimized out>, extra=<optimized out>) at sig.c:565
MPT: #5  <signal handler called>
MPT: #6  0x00002aaaef3f1387 in raise () from /lib64/libc.so.6
MPT: #7  0x00002aaaef3f2a78 in abort () from /lib64/libc.so.6
MPT: #8  0x00002aaaeeb6d369 in __gnu_cxx::__verbose_terminate_handler ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/vterminate.cc:95
MPT: #9  0x00002aaaeeb78923 in __cxxabiv1::__terminate(void (*)()) ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:48
MPT: #10 0x00002aaaeeb78968 in std::terminate ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:58
MPT: #11 0x00002aaaeeb78b4d in __cxxabiv1::__cxa_throw (obj=<optimized out>, 
MPT:     tinfo=0x2aaaee8aa4a0 <typeinfo for cv::Exception>, 
MPT:     dest=0x2aaaeda45eb0 <cv::Exception::~Exception()>)
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_throw.cc:95
MPT: #12 0x00002aaaed815e77 in cv::error(cv::Exception const&) [clone .cold] ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #13 0x00002aaaeda4819d in cv::error(int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, char const*, char const*, int) () from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #14 0x00002aaaed9503f8 in cv::operator==(cv::Mat const&, cv::Mat const&) ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #15 0x000000000041e70e in matsnd(cv::Mat const&, int, int) ()
MPT: #16 0x000000000041f6c7 in main ()
MPT: (gdb) A debugging session is active.
MPT: 
MPT: 	Inferior 1 [process 17147] will be detached.
MPT: 
MPT: Quit anyway? (y or n) [answered Y; input not from terminal]
MPT: Detaching from program: /proc/17147/exe, process 17147
MPT: [Inferior 1 (process 17147) detached]
MPT: Attaching to program: /proc/23792/exe, process 23792
MPT: [Thread debugging using libthread_db enabled]
MPT: Using host libthread_db library "/lib64/libthread_db.so.1".
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: (no debugging symbols found)...done.
MPT: 0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: warning: File "/nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py" auto-loading has been declined by your `auto-load safe-path' set to "$debugdir:$datadir/auto-load:/usr/bin/mono-gdb.py".
MPT: To enable execution of this file add
MPT: 	add-auto-load-safe-path /nasa/pkgsrc/toss3/2021Q2/gcc9/lib64/libstdc++.so.6.0.28-gdb.py
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: To completely disable this security protection add
MPT: 	set auto-load safe-path /
MPT: line to your configuration file "/home5/rrramir2/.gdbinit".
MPT: For more information about this security protection see the
MPT: "Auto-loading safe path" section in the GDB manual.  E.g., run from the shell:
MPT: 	info "(gdb)Auto-loading safe path"
MPT: Missing separate debuginfos, use: debuginfo-install glibc-2.17-326.el7_9.x86_64 libX11-1.6.7-4.el7_9.x86_64 libXau-1.0.8-2.1.el7.x86_64 libXext-1.3.3-3.el7.x86_64 libglvnd-1.0.1-0.8.git5baa1e5.el7.x86_64 libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.x86_64 libibverbs-41mlnx1-OFED.4.9.3.0.0.49417.x86_64 libmlx5-41mlnx1-OFED.5.0.0.4.0.50218.x86_64 libnl3-3.2.28-4.el7.x86_64 libnuma-3.0sgi-sgi716r61.rhel73.x86_64 libxcb-1.13-1.el7.x86_64
MPT: (gdb) #0  0x00002aaaaacde18c in waitpid () from /lib64/libpthread.so.0
MPT: #1  0x00002aaaab01f3b6 in mpi_sgi_system (
MPT: #2  MPI_SGI_stacktraceback (
MPT:     header=header@entry=0x7fffffffad50 "MPT ERROR: Rank 169(g:169) received signal SIGABRT/SIGIOT(6).\n\tProcess ID: 23792, Host: r147i5n3, Program: /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024\n\tMPT Version: HPE MPT 2.25  "...) at sig.c:340
MPT: #3  0x00002aaaab01f5b6 in first_arriver_handler (signo=signo@entry=6, 
MPT:     stack_trace_sem=stack_trace_sem@entry=0x2aab26fe0080) at sig.c:489
MPT: #4  0x00002aaaab01f843 in slave_sig_handler (signo=6, 
MPT:     siginfo=<optimized out>, extra=<optimized out>) at sig.c:565
MPT: #5  <signal handler called>
MPT: #6  0x00002aaaef3f1387 in raise () from /lib64/libc.so.6
MPT: #7  0x00002aaaef3f2a78 in abort () from /lib64/libc.so.6
MPT: #8  0x00002aaaeeb6d369 in __gnu_cxx::__verbose_terminate_handler ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/vterminate.cc:95
MPT: #9  0x00002aaaeeb78923 in __cxxabiv1::__terminate(void (*)()) ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:48
MPT: #10 0x00002aaaeeb78968 in std::terminate ()
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_terminate.cc:58
MPT: #11 0x00002aaaeeb78b4d in __cxxabiv1::__cxa_throw (obj=<optimized out>, 
MPT:     tinfo=0x2aaaee8aa4a0 <typeinfo for cv::Exception>, 
MPT:     dest=0x2aaaeda45eb0 <cv::Exception::~Exception()>)
MPT:     at ../../../../gcc-9.3.0/libstdc++-v3/libsupc++/eh_throw.cc:95
MPT: #12 0x00002aaaed815e77 in cv::error(cv::Exception const&) [clone .cold] ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #13 0x00002aaaeda4819d in cv::error(int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, char const*, char const*, int) () from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #14 0x00002aaaed9503f8 in cv::operator==(cv::Mat const&, cv::Mat const&) ()
MPT:    from /home5/rrramir2/local/opencv/lib64/libopencv_core.so.405
MPT: #15 0x000000000041e70e in matsnd(cv::Mat const&, int, int) ()
MPT: #16 0x000000000041f6c7 in main ()
MPT: (gdb) A debugging session is active.
MPT: 
MPT: 	Inferior 1 [process 23792] will be detached.
MPT: 
MPT: Quit anyway? (y or n) [answered Y; input not from terminal]
MPT: Detaching from program: /proc/23792/exe, process 23792
MPT: [Inferior 1 (process 23792) detached]
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT: Program /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024, Rank 134, Process 68885: Core dump on signal SIGABRT/SIGIOT(6) suppressed.
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.5) /home5/rrramir2/opencv/modules/core/src/matrix_expressions.cpp:32: error: (-5:Bad argument) One or more matrix operands are empty. in function 'checkOperandsExist'

MPT: Program /home5/rrramir2/job_scripts/cpu_jobs/1024_images/cpu_object_1024, Rank 150, Process 68901: Core dump on signal SIGABRT/SIGIOT(6) suppressed.
MPT ERROR: MPI_COMM_WORLD rank 134 has terminated without calling MPI_Finalize()
	aborting job
MPT: Received signal 9
